.gitignore

```diff
@@ -1 +1,23 @@
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
+*.o
+*.a
+*.so
```

Comment:
We don't need those c specific things i guess...

Reply:
Maybe not. I got a "standard Go gitignore". I'll remove them.